blocks/user-licenses/user-actions.js

```diff
@@ -0,0 +1,119 @@
+import execute from '../../scripts/zemax-api.js';
+import { hideModal, showSnackbar } from '../../scripts/scripts.js';
+
+function processResponse(event, data, renderViewMethod, hideActionModal,
+  successToastMessage, successResponseCode) {
+  let successCode = 204;
+  if (successResponseCode) {
+    successCode = successResponseCode;
+  }
+  if (data?.status === successCode) {
+    if (hideActionModal) {
+      hideModal(event);
+    }
+    // Re render view after the data is updated
+    if (renderViewMethod) {
+      renderViewMethod(event);
+    }
+    if (successToastMessage) {
+      showSnackbar(successToastMessage, '');
+    }
+  } else {
+    showSnackbar(data.message, 'error');
+  }
+}
+
+export async function addColleague(event, callback) {
+  event.preventDefault();
+  const formNode = event.target.parentNode;
+  const formData = new FormData(formNode);
+  formData.append('parentcustomerid_account', localStorage.getItem('parentcustomerid'));
+  const urlParams = new URLSearchParams(formData).toString();
+  const data = await execute('dynamics_add_colleague', `&${urlParams}`, 'POST');
```

Comment:
When I commented on this earlier, I meant to pass the object like `{ parentcustomerid_account: localStorage.getItem('parentcustomerid') }` as a parameter to `execute`, so not every caller has to construct the URLSearchParams.

Reply:
Done